logging/Limiting_Access_to_Logs.md

```diff
@@ -15,39 +17,86 @@ only for a specified namespace.
+
+Linking users to the back-end role of V4MCLUSTER_ADMIN_kibana_users via the Kibana security plug-in grants those users the access describe above.
+
+You can create new users and link them to these access controls by running the user.sh script.  For example, to create a user called "Chen" and link her to these access controls, submit the following command:
+
+<pre>
+/logging/bin/user.sh CREATE --user <i>username</i> --password <i>password</i> --namespace _all_ --tenant _all_
+</pre>
+
+Where: 
+
+- *username* is chen.
+- *password* is the password to WHAT?.
+
+## <a name="implement_access"></a>Implement Access Controls for a Tenant or Namespace
 
 Use the `/logging/bin/onboard.sh` script to implement logging access 
 controls for a namespace or SAS Viya tenant. The script performs these actions: 
 
-- Creates a Kibana tenant space
+- Creates a Kibana-tenant space.
 - Loads Kibana content such as visualizations and dashboards into the Kibana 
-tenant space
+tenant space.
 - Creates the access controls to limit access to only the appropriate 
-back-end data
-- Links the access controls to the Kibana tenant space
-- Creates a new user that can access only the information in this 
-Kibana tenant space (optional)
+back-end data.
+- Links the access controls to the Kibana-tenant space.
+- (Optional) Creates a new user that can access only the information in this 
+Kibana-tenant space.
 
 This is the syntax for the script:
 
-```
-/logging/bin/onboard.sh --namespace <namespace>  [--tenant <tenant>] [--user][<username>] [--password <password>]
-```
-The `namespace` parameter is required. It specifies the Kubernetes namespace to which access limitations are applied.
+<pre>
+/logging/bin/onboard.sh --namespace <i>namespace</i>  [--tenant <i>tenant</i>] [--user][<i>username</i>] [--password <i>password</i>]
+</pre>
 
-The `tenant` parameter specifies the SAS Viya tenant to which access limitations are applied.
+- *namespace* is required. It specifies the Kubernetes namespace to which access limitations are applied.
+- *tenant* specifies the SAS Viya tenant to which access limitations are applied.
+- *user* specifies to create an initial user with access to the 
+Kibana-tenant space. 
 
-The Kibana tenant space name is a combination of the namespace name and the 
-SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the Kibana tenant space name would be `mynamespace`. If you ran the script and specified `--namespace mynamespace --tenant mytenant1`, the Kibana tenant space name would be `mynamespace_mytenant1`. 
+The Kibana-tenant space name is a combination of the namespace name and the 
+SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the Kibana-tenant space name would be `mynamespace`. If you ran the script and specified `--namespace mynamespace --tenant mytenant1`, the Kibana-tenant space name would be `mynamespace_mytenant1`. 
```

Comment:
I've been trying to keep lines in our markdown files limited to 80 characters wide to make the markdown linter happy. Could we get these lines wrapped at (or around) 80?

Reply:
I presume I do that via CRLF?